app/frontend/src/components/RequestForm.vue

```diff
@@ -151,7 +151,7 @@ export default {
           v.length <= FieldValidations.ACRONYM_MAX_LENGTH ||
           `Acronym must be ${FieldValidations.ACRONYM_MAX_LENGTH} characters or less`,
         v =>
-          /^(?:[A-Z]{2,}[_]?)+[A-Z]{1,}$/g.test(v) ||
+          /^(?:[A-Z]{1,}[_]?)+[A-Z]{1,}$/g.test(v) ||
```

Comment:
This rule wasnt inline with the validation message.
ABC_D_EFG wasnt allowed
ABC_DE_FGH was allowed
seems a single character between underscores wasnt allowed.
I've changed it so it is allowed. Let me know if that should have been a requirement.
If so, i wonder if it needs to be mentioned in the validation tooltip

Reply:
Original regex requirement here: https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-65